src/OpenLoco/Paint/PaintEntity.cpp

```diff
@@ -1,30 +1,20 @@
 #include "PaintEntity.h"
+#include "../CompanyManager.h"
```

Comment:
I think this `include` isn't necessary, right?

Reply:
It is necessary for `CompanyManager::getCompanyColour`

Reply:
I'm not seeing that being used in this particular file, though.

Reply:
Ok, that was in the different file, now it's updated.